packages/vc-data/src/wallet/v2.ts

```diff
@@ -0,0 +1,50 @@
+import { CreateVCType, createSubjectContext, createContextConfig, createContext } from '../util/v2'
+
+export type CryptographicWallet = {
+  '@type': 'CryptographicWallet'
+  walletAddress: string
+  blockchain?: string
+  network?: string
+  keyType?: string
+}
+
+export type WalletPersonV2 = {
+  '@type': 'WalletPerson'
+  hasCryptographicWallet: CryptographicWallet
+}
+
+export type VCWalletPersonV2Type = 'WalletCredentialPersonV2'
+
+export const getVCWalletPersonV2ContextConfig = () => {
+  const cryptoWalletContext = createSubjectContext<CryptographicWallet>({
+    type: 'CryptographicWallet',
+    base: 'bloomSchema',
+    properties: {
+      walletAddress: 'bloomSchema',
+      blockchain: 'bloomSchema',
+      network: 'bloomSchema',
+      keyType: 'bloomSchema',
+    },
+  })
+
+  const walletPersonContext = createSubjectContext<WalletPersonV2>({
+    type: 'WalletPerson',
+    base: 'bloomSchema',
+    properties: {
+      hasCryptographicWallet: 'bloomSchema',
+    },
+  })
+
+  return createContextConfig<VCWalletPersonV2Type>({
+    type: 'WalletCredentialPersonV2',
```

Comment:
So our credential will be WalletCredentialPerson? What does the cred represent? 
1. They own this wallet
2. They own the Wallet that meets the rescission reqs

Reply:
`OwnedWalletCredential` ?